pages/services.js

```diff
@@ -26,7 +26,7 @@ const ImageContainer = styled.div`
   padding: 0.5rem;
 `;
 
-const MyImage = ({ src, alt }) => {
+const ServicesImage = ({ src, alt }) => {
   const X = 3;
   return <Image src={src} alt={alt} width={2035 / X} height={1570 / X} />;
 };
```

Comment:
Next i'm make this into its own component, outside of here, and just import, but good for now!